Automattic/Co-Authors-Plus
# Author select improvements

## Description

This PR makes some improvements to author selection box.

- Use debounce to delay author search request
- Fix long author names overlapping with icons

Should include any special considerations, decisions, and links to relevant GitHub issues.

Please don't include internal or private links

## Deploy Notes

No dependencies.

## Steps to Test

Outline the steps to test and verify the PR here.

- Monitor author search request after search keyword is inputted
- Add a guest author with very long name, it shouldn't overlap with the move and close icons in author select panel
